examples/06-verif-manual/vm-008-parametric_calculation.py

```diff
@@ -0,0 +1,216 @@
+            x2, y2, z2 = -200, 25, 80
+            dist_kp = math.sqrt((x2 - x1)**2 + (y2 - y1)**2 + (z2 - z1)**2)
+            print(dist_kp)
+
+"""
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL and import Numpy and Pandas libraries.
+
+# sphinx_gallery_thumbnail_path = '_static/vm8_setup.png'
+
+import numpy as np
+import pandas as pd
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start MAPDL.
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Pre-Processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.clear()
+mapdl.verify()
+_ = mapdl.prep7()
+
+
+###############################################################################
+# Define Class
+# ~~~~~~~~~~~~
+# Identifying the class ``create`` with methods ``create_kp_method`` and
+# ``create_node_method`` to calculate and plot the distances between keypoints
+# and nodes.
+
+class Create:
+    def __init__(self, x1, y1, z1, x2, y2, z2):
+        # Coordinate Attributes.
+        self.x1 = x1
+        self.y1 = y1
+        self.z1 = z1
+        self.x2 = x2
+        self.y2 = y2
+        self.z2 = z2
+
+    def create_kp_method(self):
+
+        # Define keypoints by coordinates.
+        kp1 = mapdl.k(npt=3, x=self.x1, y=self.y1, z=self.z1)
+        kp2 = mapdl.k(npt=4, x=self.x2, y=self.y2, z=self.z2)
+
+        # Get the distance between keypoints.
+        dist_kp, kx, ky, kz = mapdl.kdist(kp1, kp2)
+
+        # Plot keypoints.
+        mapdl.kplot(show_keypoint_numbering=True,
+                    vtk=True,
+                    background="grey",
+                    show_bounds=True,
+                    font_size=26)
+        return dist_kp, mapdl.klist()
+
+    def create_node_method(self):
+
+        # Define nodes by coordinates.
+        node1 = mapdl.n(node=1, x=self.x1, y=self.y1, z=self.z1)
+        node2 = mapdl.n(node=2, x=self.x2, y=self.y2, z=self.z2)
+
+        # Get the distance between nodes.
+        dist_node, node_x, node_y, node_z = mapdl.ndist(node1, node2)
+
+        # Plot nodes.
+        mapdl.nplot(nnum=True,
+                    vtk=True,
+                    color="grey",
+                    show_bounds=True,
+                    font_size=26)
+        return dist_node, mapdl.nlist()
+
+
+###############################################################################
+# Distance between keypoints
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Using already created method for keypoints to get the distance between them
+# and print out an output. The keypoints have got next coordinates:
+#
+# * :math:`K_{\mathrm{3(x,y,z)}} = 100, 0, 30`
+# * :math:`K_{\mathrm{4(x,y,z)}} = -200,25,80`
+
+kp = Create(100, 0, 30, -200, 25, 80)
+kp_dist, keypoint_list = kp.create_kp_method()
+print(f"Distance between keypoint is: {kp_dist:.2f}\n\n"
+      f"{keypoint_list}")
```

Comment:
If you want to plot and also print ``KLIST``, you could do those as methods inside ``Create`` class. But I do believe it is not needed. Since:

```suggestion
kp_dist = kp.create_kp_method()
mapdl.kplot()
print(f"Distance between keypoint is: {kp_dist:.2f}\n\n")
print(mapdl.klist())
```
This works.